mtui/template/testreport.py

```diff
@@ -430,6 +431,14 @@ def refhosts_from_tp(self, testplatform):
     def list_bugs(self, sink, arg):
         return sink(self.bugs, self.jira, arg)
 
+    def _get_result_links(self):
+        incident_id = self.repository.rstrip("/").split("/")[-1]
+        dboard_burl = "http://dashboard.qam.suse.de/incident/"
+        openqa_burl = "https://openqa.suse.de/tests/overview?build=:"
```

Comment:
Please avoid specifying any SUSE internal URLs that are not relevant to this "upstream" open source project

Reply:
So should I avoid this change entirely?
or get the base urls from some other ways like the user environment?
or have them in mtui/config.py where a few simillar urls are already defined?

Reply:
reading from user environment or mtu/config.py both sounds good. I would just avoid specifying any SUSE internal URLs but for convenience maybe it makes sense to provide them but then please in a specific config file. https://github.com/openSUSE/mtui/blob/main/mtui/config.py#L114 looks similar so I guess it's ok if you define URLs there.